cg/models/scout/scout_load_config.py

```diff
@@ -180,6 +182,7 @@ class NalloLoadConfig(ScoutLoadConfig):
     samples: list[ScoutNalloIndividual] = []
     somalier_samples: str | None = None
     somalier_pairs: str | None = None
+    str_catalog: str | None = None
```

Comment:
Same here, this is part of the Reviewer class, and set on the individual (`reviewer.catalog`); compare e.g. https://github.com/Clinical-Genomics/scout/blob/5e4d99395eba4367030c5f74eb8bab4f2f86f087/docs/admin-guide/load-config.md?plain=1#L84.

Reply:
I'm not the person to ask, really: I would have asked you since you made the Hermes PR and thus defined the tag names! 😸  But I see, seems the naming in the class in cg is different from the tags, and from the scout config file key as well. So maybe this is what you are are looking for? It seems cg has some kind of case level str_catalog as well - dont know what that is about. There is nothing like that in Scout. 😊 
https://github.com/Clinical-Genomics/cg/blob/d6d8707714398fdaf43b9b539a085f98ff1d55aa/cg/meta/upload/scout/hk_tags.py#L90